### PR TITLE
Dac service + misc

### PR DESCRIPTION
**Pull request by iaj2**
- #38 
- Modifies ADC types and modifies integration tests to accommodate
- Adds singleSample() ADC method and test
- Changed some service method names that were mistyped as snake_case but meant to be camelCase
- Changed logs for services

### CODE REVIEW

**Comment by iaj2** [comment]
Since runner is broken and integration tests are down, these are results from my edgepi:
```
Test Suites: 8 passed, 8 total
Tests:       117 passed, 117 total
Snapshots:   0 total
Time:        22.52 s, estimated 48 s
Ran all test suites matching /tests\/integration\//i.
```

File: src/services/DacService/DacService.ts [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+import path from 'path'
+import { RpcChannel } from '../../rpcChannel/RpcChannel'
+import type { serverResponse, serviceRequest } from '../../rpcChannel/ReqRepTypes'
+import type { DACChannel, GainStateMsg, GetState, State, StateMsg } from './DacTypes'
+import { SuccessMsg } from '../serviceTypes/successMsg'
+
+const protoPckgPath = path.join(require.resolve('@edgepi-cloud/rpc-protobuf'), '..');
+
+/**
+ * @constructor DinService class for calling EdgePi digital input SDK methods through RPC
```
Comment (sjpark608) [inline src/services/DacService/DacService.ts:11]: DinService -> DacService?

**Review by sjpark608 — APPROVED** [review]

File: src/services/RelayService/RelayService.ts [hunk 2]
```diff
@@ -35,7 +35,11 @@ class RelayService {
       requestMsg: {/* Empty Msg */}
     }
     // Call method through rpc
-    console.debug("Sending  request through rpcChannel")
+    console.info(
+      `Calling Relay ${methodName.replace(/_([a-z])/g, (_, group1) => group1.toUpperCase())}`+
+     ` through Rpc Channel..`
+     )
```
Comment (naomiven) [inline src/services/RelayService/RelayService.ts:41]: so the original `method_name` is snake_case but you're logging it as camelCase? why?

**Review by naomiven — COMMENTED** [review]
looks good so far

File: src/services/DacService/DacService.ts (hunk 1 above)
Comment (iaj2) [reply src/services/DacService/DacService.ts:11]: lmao yea typo

File: src/services/RelayService/RelayService.ts (hunk 2 above)
Comment (iaj2) [reply src/services/RelayService/RelayService.ts:41]: basically on the python server side it expects to get `method name` in snake_case but on this javascript client, camelCase is convention. So I think logging out the method name in snake_case is weird when the method that is being called on this side is in camelCase

File: src/services/DacService/DacService.ts (hunk 1 above)
Comment (iaj2) [reply src/services/DacService/DacService.ts]: 2036b1a2ca3c0534e25f725af3abd849d73b37fc

**Review by naomiven — APPROVED** [review]

**Pull request merged**